scheduler/local_scd_server.py

```diff
@@ -15,81 +15,81 @@
 import time
 import argparse
 
-BOREALISPATH = os.environ['BOREALISPATH']
+BOREALISPATH = os.environ["BOREALISPATH"]
 sys.path.append(f"{BOREALISPATH}/scheduler")
 import scd_utils
 
 
-SWG_GIT_REPO_DIR = 'schedules'
+SWG_GIT_REPO_DIR = "schedules"
 SWG_GIT_REPO = "https://github.com/SuperDARN/schedules.git"
 
 EXPERIMENTS = {
     "sas": {
-        "common_time":          "twofsound",
-        "discretionary_time":   "twofsound",
-        "htr_common_time":      "twofsound",
-        "themis_time":          "themisscan",
-        "special_time_normal":  "twofsound",
-        "rbsp_time":            "rbspscan",
-        "no_switching_time":    "normalscan",
-        "interleaved_time":     "interleavedscan",
-        "normalsound_time":     "normalsound"
+        "common_time": "twofsound",
+        "discretionary_time": "twofsound",
```

Comment:
Hmm I kinda like having the right side whitespace aligned

Reply:
Me too, but this fixes the problem where you have one entry that is longer than the others so you have to tab all of them over an extra couple tabs